templates/includes/change_installment_period_js.html

```diff
@@ -0,0 +1,20 @@
+<script>
+  $('input[name="installment_period"]').click(function() {
```

Comment:
If you totally wanted to ditch jQuery, you could use `document.querySelectorAll` (certainly not necessary). Also, I think we should technically be binding to the `change` event (not click), but it shouldn't really make a difference.